recipe/build.sh

```diff
@@ -12,5 +24,19 @@ cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
       -DLLVM_ENABLE_TERMINFO=OFF \
       ..
 
-make -j${CPU_COUNT}
-make install
+echo '################'
+echo $CC
+echo $CXX
+ccache -s
+echo '################'
+
+bash -c '#!/bin/sh
```

Comment:
This was a quick hack to avoid timeout. If the build times out, the cache is not uploaded, so we need to fail earlier. This is only necessary on long builds and could be added to each recipe.

We can replace this with the ```timeout``` command provided by ```coreutils```:
```
timeout 2400 make -j${CPU_COUNT}
```

The problem is that it is only available on OSX through homebrew.